test_app/lib/screens/join_screen.dart

```diff
@@ -437,13 +435,13 @@ class _JoinConferenceContentState extends State<JoinConferenceContent> {
 
   Future<Conference> createConference() async {
     var options = conferenceCreateOptions();
+    
     String alias = options.alias != null ? options.alias! : "";
     var subscription =
         SubscriptionType.values.map((e) => Subscription(e, alias)).toList();
     await _dolbyioCommsSdkFlutterPlugin.notification.subscribe(subscription);
 
-    var conference = _dolbyioCommsSdkFlutterPlugin.conference.create(options);
-    return conference;
+    return await _dolbyioCommsSdkFlutterPlugin.conference.create(options);
```

Comment:
How this changes are releated with this task ?

Reply:
It is not I was fixing a couple of things in the app as well.